recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./configure --prefix=$PREFIX
+./configure --prefix=$PREFIX PERL='/usr/bin/env perl'
```

Comment:
Also not that anaconda is doing the right thing with the shebang. It adds the place holder `#! /opt/anaconda1anaconda2anaconda3/bin/perl` and then replaces with the env `$PREFIX`.